config/routes.rb

```diff
@@ -9,4 +9,5 @@
   get "instruments/:id", to: "instruments#show", as: :instrument
   get "/instruments/:id/rent/new", to: "bookings#new", as: :bookings
   post "/instruments/:id/rent/new", to: "bookings#create"
+  delete "/booking/:id/cancel", to: "bookings#destroy"
```

Comment:
maybe define the destroy action with a path name?

as: :destroy ?

Like that we can link to it from wherever the user can cancel/delete their bookings?